src/templates/JavaScript Helper 2/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "{{name}}",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "start": "parcel index.html --port 3000",
+    "build": "parcel build index.html",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix"
+  },
+  "dependencies": {
+    "algoliasearch": "3.30.0",
```

Comment:
not sure what's causing this not to be updated, but 3.35.1 is the latest release

Reply:
I didn't bump all `dependencies`, we can let Renovate do that since we're more confident in those changes than `devDependencies`.